scripts/index.js

```diff
@@ -0,0 +1,182 @@
+const mainSkills = document.getElementById("main-skills");
```

Comment:
I recommend to use useSelector with '#' to get elements by id as this way is more modern nowodays. Such methods as .getElementById are deprecated.